### PR TITLE
[1454][ADD] sale_line_note

### PR DESCRIPTION
**Pull request by TimLai125**
#[1454](https://www.quartile.co/web?debug=#id=1454&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: sale_line_note/views/sale_order_views.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="name">sale.order.form</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']//tree//field[@name='name']" position="after">
```
Comment (yostashiro) [inline sale_line_note/views/sale_order_views.xml]:
```suggestion
            <xpath expr="//field[@name='order_line']//tree//field[@name='price_tax']" position="after">
```
Customer requests that the field show at the end of the tree fields.

File: sale_line_note/i18n/ja.po [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_line_note
+#: model:ir.model.fields,field_description:sale_line_note.field_sale_order_line__delivery_note
+msgid "Delivery Note"
+msgstr "配送備考"
```
Comment (yostashiro) [inline sale_line_note/i18n/ja.po]:
```suggestion
msgstr "配送ノート"
```
To be consistent with the field name in stock.move.

**Review by yostashiro — CHANGES_REQUESTED** [review]

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Comment by TimLai125** [comment]
The TravisCI build fails for the pre-commit (the issue cannot be reproduced on the development, I will open another ticket to follow-up)

**Pull request merged**